Jenkinsfile

```diff
@@ -177,6 +191,15 @@ pipeline {
                                             }
                                         } catch(Exception err) {
                                         }
+                                        try {
+                                            wrap([$class: 'BuildUser']) {
+                                                dir('scylla-cluster-tests') {
+                                                    runRestoringMonitoringStack()
+                                                }
+                                            }
+                                        } catch(Exception err) {
+                                            currentBuild.result = 'FAILURE'
```

Comment:
in all the catch statments, please print the exception, it's now failing in Jenkins and we can't understand why
you code not yet running

Reply:
Added